repose-aggregator/extensions/api-validator/src/main/java/org/openrepose/filters/apivalidator/ValidatorConfigurator.java

```diff
@@ -135,6 +136,11 @@ private String getPath(String path, String configRoot) {
     }
 
     private String getWadlPath(String uri, String configRoot) {
-        return new File(configRoot, uri).toURI().toString();
+        //If the wadl path is already absolute, just return that rather than prepending the config root
+        if(Paths.get(uri).isAbsolute()) {
```

Comment:
Either works I suppose. I didn't parse the documentation for URI correctly, and ended up down the `Paths` file path. Do I need to change it?

```
scala> import java.net.URI
import java.net.URI

scala> val configRoot = new URI("file:///etc/repose")
configRoot: java.net.URI = file:///etc/repose

scala> configRoot.resolve("/something")
res0: java.net.URI = file:/something

scala> configRoot.resolve("something")
res1: java.net.URI = file:/etc/something

scala> val configRoot2 = new URI("file:///etc/repose/")
configRoot2: java.net.URI = file:///etc/repose/

scala> configRoot2.resolve("/something")
res2: java.net.URI = file:/something

scala> configRoot2.resolve("something")
res3: java.net.URI = file:/etc/repose/something
```